schemas/ufo/fontinfo.fbs

```diff
@@ -64,7 +66,7 @@ table FontInfo {
   openTypeNameRecords: [OpenTypeNameRecord];
 
   /// https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/#opentype-os2-table-fields
-  openTypeOS2WidthClass: uint;
```

Comment:
good that you defined an enum for WidthClass. However this is the only field right now that has a default value defined, so it seems a bit out of place. There needs to be a more global decision as to what to do with defaults

Reply:
Thank you for the comments. 
So here the problem is I will get an error for not setting the default value, probably because I didn't define "0" in enums. So, defining zero or setting a default value may be needed here.